hhs_oauth_server/settings/base.py

```diff
@@ -413,6 +413,10 @@
 SESSION_COOKIE_SECURE = env('DJANGO_SECURE_SESSION', True)
 SESSION_EXPIRE_AT_BROWSER_CLOSE = True
 
+APPLICATION_TEMPORARILY_INACTIVE = """This application ({}) is temporarily inactive.
+        If you are the app maintainer, please contact the Blue Button 2.0 API team.
+        If you are a Medicare Beneficiary and need assistance,
+        please contact the application support or call 1-800-MEDICARE (1-800-633-4227)"""
```

Comment:
@JFU-GIT This should be refactored so that the detail message is contained in a single line. Noticed that it contains new lines "\n" characters in the resulting detail message.

```suggestion
APPLICATION_TEMPORARILY_INACTIVE = "This application ({}) is temporarily inactive. " \
                                   "If you are the app maintainer, please contact the Blue Button 2.0 API team. " \
                                   "If you are a Medicare Beneficiary and need assistance, " \
                                   "please contact the application support or call 1-800-MEDICARE (1-800-633-4227)"
```